test/exe/Main.hs

```diff
@@ -1984,6 +1984,8 @@ cradleTests = testGroup "cradle"
     ,testGroup "loading" [loadCradleOnlyonce]
     ]
 
+{-# ANN loadCradleOnlyonce ("HLint: ignore Redundant bracket" :: String) #-}
```

Comment:
Note the recommended ignore is `{- HLINT ignore Redundant bracket -}` as that won't cause a Template Haskell overhead parsing the Ann. I will fix the bug though!

Reply:
Thanks fixed! I can never remember the syntax. And thanks for opening an issue. I was just about to report it but clearly you beat me to it :slightly_smiling_face: